devolutions-session/src/dvc/task.rs

```diff
@@ -830,6 +841,49 @@ async fn handle_exec_error(dvc_tx: &WinapiSignaledSender<NowMessage<'static>>, s
     }
 }
 
+fn set_kbd_layout(layout: SetKbdLayoutOption) -> anyhow::Result<()> {
+    // SAFETY: No outstanding preconditions.
```

Comment:
Mostly for uniformity:
```suggestion
    // SAFETY: FFI call with no outstanding preconditions.
```